src/summary/documentation/dimensions-card.component.tsx

```diff
@@ -86,7 +87,9 @@ export default function DimensionsCard(props: DimensionsCardProps) {
           </table>
         </SummaryCardRowContent>
       </SummaryCardRow>
-      <ShowMoreCard func={() => setShowMore(true)} />
+      <SummaryCardFooter linkTo={`/patient/${patientUuid}/chart/dimensions`}>
+        <p className="omrs-bold">See all</p>
```

Comment:
As it's always going to be See All, why not just include this (line 91) in the SummaryCardFooter itself?